broker_test.go

```diff
@@ -55,11 +55,6 @@ func TestBrokerRedisSend(t *testing.T) {
 			continue
 		}
 		messageList := messageJSON.([]interface{})
-		if string(messageList[0].([]byte)) != "celery" {
-			t.Errorf("test '%s': non celery message received", tc.name)
-			releaseCeleryMessage(celeryMessage)
-			continue
-		}
```

Comment:
Add additional tests to support setting queue name

Reply:
There are 2 new tests cases with new broker in `gocelery_test.go` in second commit, I can check the queue here, but as mentionned in the commit log this will alway match the queue we BLPOP from. We do pop from a single redis list for now, and you already do check we had a returned value before that. Did you have other kind of test in mind ?